docs/source/users/options/fitting_option.rst

```diff
@@ -13,8 +13,8 @@ Software (:code:`software`)
 Software is used to select the fitting software to benchmark, this should be
 a newline-separated list. Available options are:
 
-* ``bumps`` (default software)
-* ``dfo`` (default software)
+* ``bumps`` (external software)
```

Comment:
Can you link these to the external instructions section (the ref bit on the other lines)

It's this [page](https://fitbenchmarking.readthedocs.io/en/latest/users/install_instructions/fitbenchmarking.html#extra-dependencies) and just tells people how to install them.